### ISSUE TITLE
Rev 2 on OSHpark?

### ISSUE THREAD

**Issue by manuti** [washingtonian88/onion2-breakout#1]
Are you planing to upload the Rev 2 to OSHpark?
Thanks

**Comment by washingtonian88** [comment]
Just uploaded it to [OSHpark here](https://oshpark.com/shared_projects/nRecWv9b) - thanks for asking! Iv'e been having an issue getting the additional boards I ordered from indigogo and was putting off another order until they had shipped.

Hope you find the board useful :+1: 

Best,
-Josef

**Comment by manuti** [comment]
Thanks, I order [a different board before you release rev1](https://oshpark.com/shared_projects/GOi9e1ZF)
Now I'm waiting for the materials.
But for me your board wins for the addition of the USB connector.
Is possible to add a BOM for your board?

**Comment by washingtonian88** [comment]
No problem - I had a BOM published in the *.xlsx file. I have made a *.csv copy of this ( onion2 breakout.csv ) and have also updated the OSHpark project to have the build. 

I really wanted the USB connector as well (hence the board). I used Jensa's board as inspiration, they are really quite similar, though didn't see any links to his project and thought I would share my own.  

Best,
-Josef
